Telemetry/TelemetryClientReceiver.cs

```diff
@@ -0,0 +1,94 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+extern alias Client;
+
+using System;
+using System.Collections.Generic;
+using Client::Microsoft.Applications.Events;
+
+namespace TelemetryReceivers
```

Comment:
We should probably re-consider the namespace:
perhaps: Microsoft.Identity.Client.Telemetry

Reply:
Sure, changed it to Microsoft.Identity.Client.Telemetry

Reply:
I'm not seeing this change

---
In reply to: [234785661](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1412#discussion_r234785661) [](ancestors = 234785661)

Reply:
Should be there now. After the conversation offline, I also made changes to some of the other class names: changed namespace to Microsoft.Identity.Client.DevAppsTelemetry, and changed the "Receiver" classes to  ClientTelemetryHandler and ServerTelemetryHandler . I don't have a strong opinion either way, so if anybody has suggestions for better names I'm happy to change them.